src/dotnet/CarbonAware.WebApi.Tests/unitTests/CarbonAwareControllerTests.cs

```diff
@@ -0,0 +1,79 @@
+namespace CarbonAware.WepApi.UnitTests;
+
+using System.Collections.Generic;
+using CarbonAware.Model;
+using CarbonAware.WebApi.Controllers;
+using Microsoft.AspNetCore.Mvc;
+using NUnit.Framework;
+
+/// <summary>
+/// Testing Web API Controller
+/// </summary>
+[TestFixture]
+public class CarbonAwareControllerTests : TestsBase
+{   
+    /// <summary>
+    /// Tests that successfull call to plugin results in action with 200 status
```

Comment:
If this is using the static data provider, maybe we can specify what data payload the test expects to be there